sbol_utilities/default_components.py

```diff
@@ -0,0 +1,148 @@
+from typing import List, Tuple
+
+import sbol3
+import tyto
+
+def promoter(name: str, description: str, sequence: str)-> Tuple[sbol3.Component, sbol3.Sequence]:
+    """
+    Creates a promoter and a sequence Components with a given name, description, and sequence.
+    """
+    promoter_seq = sbol3.Sequence(f'{name}_seq')
+    promoter_seq.elements= sequence
+    promoter_seq.encoding = 'https://identifiers.org/edam:format_1207'
+
+    promoter = sbol3.Component(name, sbol3.SBO_DNA)
+    promoter.name = name
+    promoter.description = description
+    promoter.roles. append(sbol3.SO_PROMOTER)
+    promoter.sequences.append(promoter_seq)
```

Comment:
These could be done in a single constructor call. For example (this might be a little off syntactically so you may have to adjust it, I'm typing from memory):
```python
promoter = sbol3.Component(name, sbol3.SBO_DNA,
                           name=name,
                           description=description,
                           roles=[sbol3.SO_PROMOTER],
                           sequences=[promoter_seq])
```

Reply:
Thanks @tcmitchell this helped me to implement kwargs.